src/Microsoft.DotNet.Cli.Utils/StreamForwarder.cs

```diff
@@ -7,45 +7,45 @@ namespace Microsoft.DotNet.Cli.Utils
 {
     public sealed class StreamForwarder
     {
-        private const int DefaultBufferSize = 256;
-
-        private readonly int _bufferSize;
         private StringBuilder _builder;
         private StringWriter _capture;
         private Action<string> _write;
         private Action<string> _writeLine;
 
-        public StreamForwarder(int bufferSize = DefaultBufferSize)
+        public string CapturedOutput
         {
-            _bufferSize = bufferSize;
+            get 
+            {
+                return _capture?.GetStringBuilder()?.ToString();
+            }
         }
 
-        public void Capture()
+        public StreamForwarder Capture()
         {
             if (_capture != null)
             {
                 throw new InvalidOperationException("Already capturing stream!");
             }
             _capture = new StringWriter();
-        }
 
-        public string GetCapturedOutput()
-        {
-            return _capture?.GetStringBuilder()?.ToString();
+            return this;
         }
 
-        public void ForwardTo(Action<string> write, Action<string> writeLine)
+        public StreamForwarder ForwardTo(Action<string> writeLine)
         {
             if (writeLine == null)
             {
                 throw new ArgumentNullException(nameof(writeLine));
             }
+
             if (_writeLine != null)
```

Comment:
Can you encapsulate this in a method? Like WriteLineAlreadySet? I was asking myself how this was going to work. I had to read this like twice to notice the _.